androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/StopDetailsFilteredRouteView.kt

```diff
@@ -23,121 +26,113 @@ import com.mbta.tid.mbta_app.android.component.RouteHeader
 import com.mbta.tid.mbta_app.android.util.fromHex
 import com.mbta.tid.mbta_app.model.Alert
 import com.mbta.tid.mbta_app.model.RealtimePatterns
-import com.mbta.tid.mbta_app.model.Route
-import com.mbta.tid.mbta_app.model.StopDetailsDepartures
 import com.mbta.tid.mbta_app.model.StopDetailsFilter
 import com.mbta.tid.mbta_app.model.TripDetailsFilter
-import com.mbta.tid.mbta_app.model.TripInstantDisplay
-import com.mbta.tid.mbta_app.model.UpcomingTrip
 import com.mbta.tid.mbta_app.model.response.GlobalResponse
 import kotlinx.datetime.Instant
 
 @Composable
 fun StopDetailsFilteredRouteView(
-    departures: StopDetailsDepartures,
+    viewModel: StopDetailsViewModel,
```

Comment:
This diff looks dramatic - just swapping viewModel and wrapping the body in `if(departures != null)`